ddl/job_table.go

```diff
@@ -251,39 +256,27 @@ func (d *ddl) startDispatchLoop() {
 	defer ticker.Stop()
 	var ok bool
 	for {
+		if !d.isOwner() {
+			log.Warn("[ddl] not owner, continue dispatch loop")
```

Comment:
this log is not allowed, there is only one owner in a cluster and we will get a lot of log

Reply:
Removed it.